### PR TITLE
Add Russian translations

### PR DESCRIPTION
**Pull request by v1docq47**
Added full translation of Zero to Monero: Second Edition into Russian
Link to translation in Overleaf - https://www.overleaf.com/read/gvdnrxqgmksy

### CODE REVIEW

File: translations/ru/main.tex [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+  \includegraphics[width=3cm]{front/figures/monero-symbol-1280.png}
+\end{center}
+
+
+\vfill{}
+{\par\centering \textsc{\huge От нуля к Monero: Второе издение}\par}
+\bigskip \bigskip
+{\par\centering \textsc{\LARGE Техническое руководство по приватной цифровой валюте для начинающих, любителей и экспертов}\par}
+\bigskip
+{\par\centering \textsc{\LARGE Опубликовано 4 апреля 2004 г. (v2.0.0)}\par}
```
Comment (UkoeHB) [inline translations/ru/main.tex:174]: 2004 -> 2020

File: translations/ru/main.tex [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+%{\par\centering \textsc{\LARGE \today{}, Pre-Publication Draft 1.2.0}\par}
+\bigskip
+{\par\centering \textsc{\large koe\footnote{ukoe@protonmail.com}, Курт M. Алонсо\footnote{kurt@oktav.se}, Саранг Ноезер\footnote{sarang.noether@protonmail.com}}%\\(for questions and comments, please CC both authors)
```
Comment (UkoeHB) [inline translations/ru/main.tex:177]: Do you want to add 'translated by'?

**Review by UkoeHB — CHANGES_REQUESTED** [review]

File: translations/ru/main.tex (hunk 1 above)
Comment (v1docq47) [reply translations/ru/main.tex]: Thanks! Fixed!

File: translations/ru/main.tex (hunk 2 above)
Comment (v1docq47) [reply translations/ru/main.tex]: Added \linebreak Перевод выполнен v1docq47\footnote{chiptune@protonmail.ch}

**Pull request merged**